### PR TITLE
Orion 2 integration

### PR DESCRIPTION
**Pull request by drillprop**
PR was created for the Orion v2 test purposes.

Tracking issue https://github.com/Joystream/atlas/issues/3630

### CODE REVIEW

File: packages/atlas/codegen.config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 overwrite: true
 
-schema: ${VITE_PRODUCTION_ORION_URL}
+schema: https://194.233.167.225.nip.io/orion/graphql
```
Comment (WRadoslaw) [inline packages/atlas/codegen.config.yml]: I think it might be good to keep the URL in `.env`

File: packages/atlas/src/api/client/index.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import { ApolloClient, ApolloLink, FetchResult, HttpLink, Observable, split } from '@apollo/client'
-import { BatchHttpLink } from '@apollo/client/link/batch-http'
-import { WebSocketLink } from '@apollo/client/link/ws'
+// import { BatchHttpLink } from '@apollo/client/link/batch-http'
```
Comment (WRadoslaw) [inline packages/atlas/src/api/client/index.ts]: To removal

File: packages/atlas/src/api/client/index.ts [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+import { GraphQLWsLink } from '@apollo/client/link/subscriptions'
 import { getMainDefinition } from '@apollo/client/utilities'
+import { createClient } from 'graphql-ws'
 
 import { ORION_GRAPHQL_URL, QUERY_NODE_GRAPHQL_SUBSCRIPTION_URL } from '@/config/env'
 
 import cache from './cache'
 
-const BATCHED_QUERIES = ['GetBasicVideos', 'GetDistributionBucketsWithBags', 'GetStorageBucketsWithBags']
+// const BATCHED_QUERIES = ['GetBasicVideos', 'GetDistributionBucketsWithBags', 'GetStorageBucketsWithBags']
```
Comment (WRadoslaw) [inline packages/atlas/src/api/client/index.ts]: Same as above

File: packages/atlas/src/api/client/index.ts [hunk 4]
```diff
@@ -25,35 +26,36 @@ const delayLink = new ApolloLink((operation, forward) => {
...
-  ])
-
-  const orionSplitLink = split(
-    ({ operationName }) => {
-      return BATCHED_QUERIES.includes(operationName)
-    },
-    batchedOrionLink,
-    orionLink
-  )
+  // todo batching not working with new orion?
```
Comment (WRadoslaw) [inline packages/atlas/src/api/client/index.ts]: As I remember batching won't be a part of Orion v2

File: packages/atlas/src/api/hooks/channel.ts [hunk 5]
```diff
@@ -18,71 +20,76 @@ import {
...
       ...variables,
       where: {
-        ...channelFilter,
         ...variables?.where,
       },
     },
   })
   return {
-    channels: data?.channels,
+    channels: data?.extendedChannels.map((extendedChannel) => extendedChannel),
```
Comment (WRadoslaw) [inline packages/atlas/src/api/hooks/channel.ts]: Why map it?

File: packages/atlas/src/api/hooks/channel.ts [hunk 6]
```diff
@@ -167,14 +194,21 @@ export const useDiscoverChannels = (
     variables: {
       ...variables,
       where: {
-        ...channelFilter,
-        activeVideosCounter_gt: 0,
+        channel: {
+          ...channelFilter,
```
Comment (WRadoslaw) [inline packages/atlas/src/api/hooks/channel.ts]: This repeats a few times, I'm wondering whether we have to spread it and allocate a new object if we don't need to provide extra params.

File: packages/atlas/src/providers/notifications/notifications.manager.tsx [hunk 7]
```diff
@@ -1,17 +1,13 @@
-import { FC, useEffect } from 'react'
-
-import { atlasConfig } from '@/config'
-
-import { useNotifications } from './notifications.hooks'
+import { FC } from 'react'
 
 export const NotificationsManager: FC = () => {
-  const { startPolling, stopPolling } = useNotifications()
+  // const { startPolling, stopPolling } = useNotifications()
```
Comment (WRadoslaw) [inline packages/atlas/src/providers/notifications/notifications.manager.tsx]: Comments

File: packages/atlas/src/views/viewer/ChannelView/ChannelViewTabs/ChannelNfts.tsx [hunk 8]
```diff
@@ -39,9 +39,15 @@ export const ChannelNfts: FC<ChannelNftsProps> = ({
...
     where: {
       ...ownedNftWhereInput,
-      creatorChannel: {
-        id_eq: channelId,
+      owner: {
+        channel: {
+          id_eq: channelId,
+        },
       },
+      // check two times if its working
```
Comment (WRadoslaw) [inline packages/atlas/src/views/viewer/ChannelView/ChannelViewTabs/ChannelNfts.tsx]: Comment

File: packages/atlas/src/utils/logs/asset.ts [hunk 9]
```diff
@@ -23,6 +23,7 @@ type StorageEvent = {
 
 export type DistributorEventEntry = {
   dataObjectId: string
+  // @ts-ignore todo fix this later
```
Comment (WRadoslaw) [inline packages/atlas/src/utils/logs/asset.ts]: Remainder

**Review by attemka — APPROVED** [review]

**Review by WRadoslaw — APPROVED** [review]

**Pull request merged**